index.html

```diff
@@ -363,6 +363,10 @@ <h5 class="page_h5">Mylyn - TaskAttributeDiff</h5>
 
     <a id="custom_a" href="https://github.com/Tasktop/code.modifications/raw/aed7ab7108e18059fd5d5bec8beb08cb22f1b37e/src/epl/SyncTaskAttributeDiff.java">Source</a><br/>
     <a id="custom_a" href="https://github.com/Tasktop/code.modifications/raw/0ea665a3ee32a6557037fd4943b5b936c97d10f7/src/about/epl-v10.txt">Eclipse Public License 1.0</a>
+
+    <h5 class="page_h5">Mylyn - WikiText</h5>
```

Comment:
I think we should put Wikitext at the top of the page, so it doesn't look buried. Especially since it's the latest addition and we may have additional visitors to the page when the email goes out. Can we put at the top a new section:

### Mylyn Wikitext

**com.planview.planview.mylyn.docs**

[Source](https://github.com/Tasktop/code.modifications/tree/master/src/wikitex)
[Eclipse Public License 2.0](https://www.eclipse.org/legal/epl-2.0/)